### PR TITLE
Add header parameter 'Authorization' for postBucket method

### PR DESCRIPTION
**Pull request by Haoning-Sun**
### What changes are proposed in this pull request?

Add header parameter 'Authorization' for postBucket method.

### Why are the changes needed?
Need to check username.

### Does this PR introduce any user facing changes?
no.

### CODE REVIEW

File: core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java [hunk 1]
```diff
@@ -352,13 +353,15 @@ public Response getBucket(@HeaderParam("Authorization") String authorization,
    */
   @POST
   @Path(BUCKET_PARAM)
-  public Response postBucket(@PathParam("bucket") final String bucket,
+  public Response postBucket(@HeaderParam("Authorization") String authorization,
```
Comment (Jackson-Wang-7) [inline core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java:356]: In my opinion, authorization is defined as final String is better.

Comment (Jackson-Wang-7) [inline core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java:356]: In my opinion, authorization is defined as final String is better.

Comment (Haoning-Sun) [reply core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java]: All finished.

**Review by Jackson-Wang-7 — APPROVED** [review]
Thanks! LGTM!

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**